lib/task/tools/purgeTask.class.php

```diff
@@ -104,7 +104,10 @@ public function execute($arguments = array(), $options = array())
       }
     }
 
-    sfContext::createInstance($this->configuration);
+    $configuration = ProjectConfiguration::getApplicationConfiguration(
```

Comment:
Wow, this is a good pattern to know. The way symfony builds the sfContext has always been pretty mysterious to me. 👍